src/cascadia/TerminalControl/TermControl.idl

```diff
@@ -7,6 +7,14 @@ namespace Microsoft.Terminal.TerminalControl
     delegate void FontSizeChangedEventArgs(Int32 width, Int32 height, Boolean isInitialChange);
     delegate void ScrollPositionChangedEventArgs(Int32 viewTop, Int32 viewHeight, Int32 bufferLength);
 
+    // c++/winrt makes it difficult to share this idl between two projects,
+    // so just pin the uuid and include it in both TermControl and App
+    [uuid("339e1a87-5315-4da6-96f0-565549b6472b")]

```

Comment:
For the record, this will make it so if we change this interface here, we'll _need_ to change the other declaration as well?

Reply:
Yeah, unfortunately.

Reply:
Good, I want the compilation error rather than the mysterious bug later

Reply:
Actually, i think having a UUID mismatch will cause a runtime failure to try_as :/

Reply:
Oh then maybe we should leave a note "if you see the runtime error, then you only updated one of the two" (and probably "If you make changes to this interface, make sure to change the other one as well")